_data/master/navbars/usage.yml

```diff
@@ -31,6 +31,8 @@ toc:
     path: /usage/routereflector/calico-routereflector
 - title: Enabling IPv6 Support
   path: /usage/ipv6
+- title: Enabling IPVS in Kubernetes
```

Comment:
For some reason this doesn't show up in the sidebar, in my rendered version.

Reply:
I am unable to reproduce it. I have found with any sidebar changes, it's best to do `make clean` and then `make serve`.

Reply:
Yep, my bad - sorry!